modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/snapshot/MajorityPeerSelect.scala

```diff
@@ -47,6 +47,7 @@ object MajorityPeerSelect {
   val minSampleSize = 20
 
   case object NoPeersToSelect extends NoStackTrace
```

Comment:
is `NoPeersToSelect` still thrown somewhere?

Reply:
It is thrown here:

https://github.com/cngo-github/tessellation/blob/20230830-prot-406/modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/snapshot/MajorityPeerSelect.scala#L68

The addition of `NoHashes` was to provide more granular insight into what went wrong.